radio/src/gui/colorlcd/menu_model.cpp

```diff
@@ -49,7 +49,7 @@ ModelMenu::ModelMenu():
   addTab(new ModelGVarsPage());
   addTab(new ModelLogicalSwitchesPage());
   addTab(new SpecialFunctionsPage(g_model.customFn));
-#if defined(LUA)
+#if defined(LUA_MODEL_SCRIPTS)
```

Comment:
If you cast your mind back to a certain `[color] mixer scripts screen` PR all will become clear... as currently the page gets adds as soon as `LUA` gets defined at all, including mere `LUA=ON` which is completely wrong, hence why trying to use the tab gives an error in all the builds... rather than being enabled by LUA_MODEL_SCRIPTS along with all the code in mixes, etc.

Reply:
`LUA_MODEL_SCRIPTS` is set in radio/src/CMakeLists.txt when -DLUA=YES. It is the way it is used in Lua and the rest of the code tree. I agree that it is not super consistent, but this is the way that it has been made.